bundles/GraphicServiceOrder/Resources/assets/multiFilesManage.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Manage order lines.
+ * @see https://symfony.com/doc/current/form/form_collections.html
+ *
+ */
+var $collectionHolder;
+var $newLinkLi = $('<li></li>');
+
+/**
+ * Setup order lines form.
+ */
+jQuery(document).ready(function () {
```

Comment:
You should use `const $ = require('jquery');` to use jQuery, but I see that we have some weird hacks to not require this in https://github.com/aakb/jira_economics/blob/develop/assets/js/app.js#L13-L16 … We'll leave this for now.

Reply:
ok